src/content/en/updates/2019/07/chrome-75-media-updates.md

```diff
@@ -0,0 +1,175 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: A round up of the audio/video updates in Chrome 75: Predicting whether playback will be smooth and power efficient for encrypted media and support of the video elements "playsInline" attribute hint.
+
+{# wf_updated_on: 2019-07-23 #}
+{# wf_published_on: 2019-07-22 #}
+{# wf_tags: news,chrome75,media #}
+{# wf_featured_image: /web/updates/images/generic/play-outline.png #}
+{# wf_featured_snippet: A round up of the audio/video updates in Chrome 75: Predicting whether playback will be smooth and power efficient for encrypted media and support of the video elements "playsInline" attribute hint. #}
+{# wf_blink_components: Blink>Media #}
+
+# Audio/Video Updates in Chrome 75 {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+Chrome's media capabilities were updated in version 75. In this article, I'll
+discuss those new features which include:
+
+- Predicting whether playback will be smooth and power efficient for encrypted
+  media.
+- Support of the video elements `playsInline` attribute hint.
+
+## Encrypted Media: Decoding Info API  {: #encrypted-media}
+
+Since Chrome 66, web developers have been able to use the [Decoding Info API] to
+query the browser about the clear content decoding abilities of the device based
+on information such as the codecs, profile, resolution, bitrates, etc. It
+indicates whether the playback will be smooth (timely) and power efficient based
+on previous playback statistics recorded by the browser.
+
+The [Media Capabilities API] specification has since been updated to handle
+encrypted media configurations as well so that websites using encrypted media
+(EME) can select the optimal media streams.
+
+In a nutshell, here’s how the Decoding Info API for EME works. Give a try to the
+[official sample].
+
+    const encryptedMediaConfig = {
```

Comment:
Shouldn't this be inside a ```` ```js ... ``` ```` block? So that it is highlighted as JavaScript?

Reply:
It doesn't need to. See https://pr-7920-dot-web-central.appspot.com/web/updates/2019/07/chrome-75-media-updates